content/3_Silver/Prefix_Sums_2.mdx

```diff
@@ -102,11 +102,94 @@ export const problems = {
 
 <FocusProblem problem={problems.maxsum} />
 
+### Solution
+
+<Spoiler title="Solution">
+
+Consider the prefix sum array. The maximum subarray sum is the maximum possible $p[i]-p[j]$ where $j\leq i$. We can iterate over each index $i$ and keep a running minimum $p[j]$ value. This yields the maximum subarray sum for each possible right bound, and the maximum over all these values is our answer.
```

Comment:
I think it would be helpful to elaborate on why this is.

Reply:
also mention that $p$ is going to be the prefix sum array

Reply:
I commit @frodakcin 's change so i think i fixed...